packages/pyml_bindgen/pyml_bindgen.0.2.0/opam

```diff
@@ -10,7 +10,7 @@ depends: [
   "dune" {>= "2.8"}
   "angstrom" {>= "0.15.0"}
   "base" {>= "v0.12"}
-  "cmdliner" {>= "1.0"}
+  "cmdliner" {>= "1.0" & with-test & < "1.1.0"}
```

Comment:
```suggestion
  "cmdliner" {>= "1.0"}
  "cmdliner" {with-test & < "1.1.0"}
```
Otherwise this means that the dependency becomes a test dependency

Reply:
Ok.